nav2_docking/opennav_docking/src/docking_server.cpp

```diff
@@ -59,6 +62,9 @@ DockingServer::on_configure(const rclcpp_lifecycle::State & state)
   get_parameter("fixed_frame", fixed_frame_);
   get_parameter("dock_backwards", dock_backwards_);
   get_parameter("dock_prestaging_tolerance", dock_prestaging_tolerance_);
+  get_parameter("initial_rotation", initial_rotation_);
```

Comment:
I'm not totally grokking the initial rotation / min angle stuff.

If you have `backward_blind` and `dock_backward` then we know we need to do the initial rotation to go from forward to reverse before doing the rest of the procedure. The `initial_rotation` would thus always be true.

Then if we know we're always rotating around, then we know we should be doing a pure 180 turn. I suppose we could also use the dock's detected position to do 180 +/- error so that we align better with it instead of a pure 180 turn though! That would be cool! But we still don't need a `min_angle` param.

Thus, unless I'm missing something, I think we don't need either of these 2 params.

Reply:
The purpose behind that was to start docking before reaching 180 degree rotation. But that's true that it breaks the concept of initial rotation to dock. I removed it.